src/itwinai/torch/distributed.py

```diff
@@ -400,7 +400,9 @@ def init(self) -> None:
                 which is already initialized.
         """
         if not distributed_resources_available():
-            raise RuntimeError("Trying to run distributed on insufficient resources.")
+            raise RuntimeError(
+                "Trying to run distributed on insufficient resources."
+            )
```

Comment:
Yeah I think it wasn't picking up the settings properly, but I think it might be better now. Probably defaulted to a shorter line length or something.